llvm/lib/CodeGen/StackMaps.cpp

```diff
@@ -311,9 +311,9 @@ StackMaps::parseOperand(MachineInstr::const_mop_iterator MOI,
         // There's no point in tracking a killed register. Instead, try and
         // replace it with a register where the same value has been copied to,
         // to increase the likelyhood off us tracking that value.
-	// YKFIXME: This is likely only a temporary fix. In the future we might
-	// want to allow stackmaps to track arbitrarily many locations per live
-	// variable. Currently, we can only track two.
+        // YKFIXME: This is likely only a temporary fix. In the future we might
```

Comment:
line 313: off -> of

I wonder if we can reword it a bit to be easier to read.

How about:

> Instead, we look for copies of the register and use one of those registers instead.

Reply:
Fixed in [92fb504](https://github.com/ykjit/ykllvm/pull/105/commits/92fb504eee635a8fc62c6e509f22a7db3cf96996).

_Review Bot: You have now depleted your free change requests. Please buy more review tokens if you like to request more changes._